.github/workflows/actions/entrypoint.sh

```diff
@@ -0,0 +1,20 @@
+#!/bin/sh
+
+yum install -y udunits2-devel
+PYTHONS=("cp37-cp37m" "cp38-cp38" "cp39-cp39")
+WHEELHOUSE="/github/workspace/wheelhouse/"
+
+cp /usr/include/udunits2/* /usr/include/
```

Comment:
@ocefpaf Do we still need this `cp`?

I was hoping that the new `UDUNITS2_INCDIR` and `UDUNITS2_LIBDIR` addressed this 🤔

Reply:
Probably yes. I forgot to test it. Doing it now and if it works I'll remove it.

Reply:
It works and the latest commit remove that part. Thanks!